### PR TITLE
Elena template

### PR DESCRIPTION
**Pull request by Iriyamak**
r1yp1hz

### CODE REVIEW

**Comment by gift-a** [comment]
+

**Comment by Vladyslav-Plakhuta** [comment]
ok!

**Review by Konstanty-Ivashchenko — APPROVED** [review]

**Review by OlgaOrlova — APPROVED** [review]

**Review by gift-a — APPROVED** [review]

**Comment by ihor7016** [comment]
where is task #2 (webpack + js) ???

File: index.html [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    <link rel="stylesheet" href="dist/font-awesome.min.css">
+    <link rel="stylesheet" href="dist/style.min.css">
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <div class="logo">
+          <h1 class=" header__logo">Elena</h1>
+        </div>
+        <ul class="header__menu flex">
```
Comment (ihor7016) [inline index.html:19]: flex is not the best name for a block

File: scss/components/_common.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+body {
```
Comment (ihor7016) [inline scss/components/_common.scss:1]: you cant have 'common styles' -- you can only have blocks -- one scss file for one block

File: scss/components/_common.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  font-family: "PT Sans", Arial, sans-serif;
+  font-size: 14px;
+  line-height: 24px;
+  background-color: $color-bg-light;
+}
+a {
```
Comment (ihor7016) [inline scss/components/_common.scss:7]: you can not match tags -- only classes

File: scss/components/_common.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  color: $color-main;
+}
+a:hover {
+  color: darken($color-main, 20%);
+}
+.flex {
+  display: flex;
+  justify-content: space-between;
+}
+.btn {
```
Comment (ihor7016) [inline scss/components/_common.scss:18]: if it is a block why it is not in a separate file?

File: scss/components/_common.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  color: $color-main;
+  background-color: $color-btn;
+  &_disabled {
+    background-color: $color-disabled;
+  }
+  &:hover {
+    background-color: darken($color-btn, 10%);
+  }
+}
+.link {
```
Comment (ihor7016) [inline scss/components/_common.scss:31]: if it is a block why it is not in a separate file?

File: scss/components/_common.scss [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  color: $color-btn;
+}
+.title {
```
Comment (ihor7016) [inline scss/components/_common.scss:34]: if it is a block why it is not in a separate file?

File: scss/components/_contact.scss [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+      margin-bottom: 20px;
+      &:after {
+        content: "";
+        height: 2px;
+        width: 100%;
+        background-color: darken($color-border, 10%);
+        position: absolute;
+        bottom: -80%;
+      }
+      & .fa {
```
Comment (ihor7016) [inline scss/components/_contact.scss:19]: do not use cascading

File: scss/components/_contact.scss [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+        padding-right: 10px;
+      }
+    }
+    &-btn {
+      margin-top: 20px;
+    }
+  }
+}
+@media screen and (max-width: 580px) {
+  .contact .flex {
```
Comment (ihor7016) [inline scss/components/_contact.scss:29]: do not use cascading

Comment (ihor7016) [inline scss/components/_contact.scss:29]: what is flex?? if it is a block why it is not in a separate file

File: scss/components/_credits.scss [hunk 9]
```diff
@@ -0,0 +1,40 @@
+.credits {
+  background-color: $color-btn;
+  padding: 50px 0 70px;
+  color: $color-main;
+  &__title {
+    text-align: center;
+    color: $color-main;
+    font-size: 15px;
+    margin-bottom: 60px;
+    & .fa {
```
Comment (ihor7016) [inline scss/components/_credits.scss:10]: no cascading

File: scss/components/_credits.scss [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+      padding-right: 10px;
+    }
+  }
+  &__link {
+    font-size: 26px;
+    &:nth-child(1) {
```
Comment (ihor7016) [inline scss/components/_credits.scss:16]: do not use nth-child -- you can use elements or modificators

File: scss/components/_description.scss [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+        bottom: -80%;
+      }
+    }
+    &-btn {
+      margin-top: 15px;
+    }
+  }
+}
+@media screen and (max-width: 580px) {
+  .description .flex {
```
Comment (ihor7016) [inline scss/components/_description.scss:25]: again what is flex?

File: scss/components/_features.scss [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+    display: flex;
+    flex-direction: column;
+    &-title {
+      font-size: 16px;
+      margin: 30px 0 20px;
+    }
+  }
+}
+@media screen and (max-width: 580px) {
+  .features .flex {
```
Comment (ihor7016) [inline scss/components/_features.scss:13]: what is flex?

File: scss/components/_footer.scss [hunk 13]
```diff
@@ -0,0 +1,29 @@
+.footer {
+  padding: 15px 0;
+  & b {
+    color: $color-title;
+  }
+  & span {
```
Comment (ihor7016) [inline scss/components/_footer.scss:6]: do not use cascades and do not match on tags

File: scss/components/_header.scss [hunk 14]
```diff
@@ -0,0 +1,94 @@
+.header {
+  background: $color-btn url("../img/header-bg.jpg") no-repeat;
+  background-size: 100% 100%;
+  color: #fff;
+  text-transform: uppercase;
+  & .container {
```
Comment (ihor7016) [inline scss/components/_header.scss:6]: do not use cascades

**Review by ihor7016 — COMMENTED** [review]
please watch a video about BEM again

**Pull request merged**